package.json

```
@@ -0,0 +1,153 @@
+{
+  "name": "terra-application",
+  "main": "lib/application-base/ApplicationBase.js",
+  "version": "1.11.0",
+  "description": "A framework to support application development with Terra components",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-application"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-application",
+    "Application",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-application/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra",
+    "overrides": [
+      {
+        "files": [
+          "src/terra-dev-site/**/*.jsx"
+        ],
+        "rules": {
+          "import/no-extraneous-dependencies": "off",
+          "import/no-unresolved": "off",
+          "import/extensions": "off"
+        }
+      },
+      {
+        "files": [
+          "*.doc.jsx"
+        ],
+        "rules": {
+          "import/no-webpack-loader-syntax": "off",
+          "import/first": "off",
+          "import/no-duplicates": "off"
+        }
+      }
+    ],
+    "rules": {
+      "react/forbid-prop-types": "off"
+    },
+    "settings": {
+      "polyfills": [
+        "Promise",
+        "Map"
+      ]
+    }
+  },
+  "stylelint": {
+    "extends": "stylelint-config-terra",
+    "rules": {
+      "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
+    }
+  },
+  "files": [
+    "lib",
+    "src",
+    "translations",
+    "tests/wdio/*-spec.js",
+    "tests/wdio/**/*-spec.js"
+  ],
+  "scripts": {
+    "clean": "rm package-lock.json; rm -rf node_modules",
```

If we aren't generating the package-lock..
```suggestion
    "clean": "rm -rf node_modules",
```